modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -573,19 +575,35 @@ You can use the `ginkgo.focus` flag to run a specific test.
 [id="cnf-performing-end-to-end-tests-running-hwlatdetect"]
 === Running hwlatdetect
 
-To perform the `hwlatdetect`, run the following command:
+.Prerequisites:
+
+* You installed the real-time kernel
+* You run the tool runs on bare hardware
+* You are logged into registry.redhat.io with your Customer Portal credentials
+
+.Procedure
+
+. Run the following command:
 
 [source, terminal]
 ----
-$ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20  registry.redhat.io/openshift4/cnf-tests-rhel8:v4.9 /usr/bin/test-run.sh -ginko.focus=”hwladetect”
+$ podman run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e \
```

Comment:
better to append `:Z ` to the volume creation as per the troubleshooting section.